### PR TITLE
Cloud hosted adjustments

### PR DESCRIPTION
**Pull request by davidwyao**
enable cloud-hosted

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -79,7 +79,18 @@ To deploy your EAR application on an Open Liberty server, run the Maven `liberty
 mvn -pl ear liberty:run
 ----
 
+ifndef::cloud-hosted[]
 Once the server is running, you can find the application at the following URL: http://localhost:9080/converter/[http://localhost:9080/converter/^]
+endif::[]
+ifdef::cloud-hosted[]
+Once the server is running, open another command-line session by selecting **Terminal** > **New Terminal**
```
Comment (dmuelle) [inline README.adoc]: Once the server is running
--->
After the server is running

File: README.adoc [hunk 2]
```diff
@@ -176,11 +194,12 @@ start/pom.xml
 ----
 include::finish/pom.xml[]
 ----
-Set the [hotspot=packaging file=0]`basic configuration` for the project. Set `pom` as the [hotspot=packagingType file=0]`packaging` element of the parent [hotspot]`pom.xml` file.
 
-In the parent [hotspot]`pom.xml` file, list all of the [hotspot=modules file=0]`modules` that you want to aggregate for the application.
+Set the [hotspot=packaging file=0]`basic configuration` for the project. Set `pom` as the [hotspot=packagingType file=0]`packaging` element of the parent [hotspot file=0]`pom.xml` file.
```
Comment (dmuelle) [inline README.adoc]: Set `pom` as the [hotspot=packagingType file=0]`packaging` element of the parent [hotspot file=0]`pom.xml` file.

How is this supposed to read? It looks a little weird

Set `pom` as the `packaging` element of the parent `pom.xml` file.

should it be

Set `pom` as the value for the `packaging` element of the parent `pom.xml` file.

File: README.adoc [hunk 3]
```diff
@@ -236,19 +271,28 @@ include::finish/jar/src/main/java/io/openliberty/guides/multimodules/lib/Convert
 
 Change the [hotspot=getFeetMethod file=1]`getFeet` method so that it converts from centimetres to feet, and the [hotspot=getInchesMethod file=1]`getInches` method so that it converts from centimetres to inches. Update the `sum`, `diff`, `product` and `quotient` functions so that they add, subtract, multiply, and divide 2 numbers respectively.
 
+ifndef::cloud-hosted[]
 Now revisit the application at the http://localhost:9080/converter/[http://localhost:9080/converter/^] URL. Try entering a height in centimetres and see if it converts correctly.
+endif::[]
+ifdef::cloud-hosted[]
+Now revisit the application at the URL you previously found. Use the following command to get the URL.
```
Comment (dmuelle) [inline README.adoc]: Now revisit the application at the URL you previously found. Use the following command to get the URL.

This seems a little confusing- it tells you to use the URL you previously found, then tells you to run a command for the URL. If they are both the same URL, might be more clear to say

Now revisit the application at the URL you previously found by running the following command.

or similar

**Review by dmuelle — COMMENTED** [review]
a few suggestions

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc]: right, it makes it clearer

File: README.adoc (hunk 1 above)
Comment (gkwan-ibm) [reply README.adoc]: updated

File: README.adoc (hunk 2 above)
Comment (gkwan-ibm) [reply README.adoc]: updated

File: README.adoc (hunk 3 above)
Comment (gkwan-ibm) [reply README.adoc]: updated

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**